tests/chainerx_tests/unit_tests/routines_tests/test_math.py

```diff
@@ -581,6 +581,58 @@ def forward_xp(self, inputs, xp):
         return xp.tanh(x),
 
 
+@op_utils.op_test(['native:0', 'cuda:0'])
+@pytest.mark.parametrize('lhs, rhs', [
+    (numpy.asarray([[1., 2.],[4., 5.]]), numpy.asarray([1., 3.])), (numpy.asarray(
+        [1., 2., 3.]), numpy.asarray([3., 4., 5.]))
+])
+@pytest.mark.parametrize('contiguous', [None, 'C'])
+class testPow(op_utils.NumpyOpTest):
+
+    def setup(self, lhs, rhs, contiguous, float_dtype):
+        self.lhs = lhs
+        self.rhs = rhs
+        self.contiguous = contiguous
+
+        if float_dtype == 'float16':
+            self.check_backward_options = {'atol': 5e-4, 'rtol': 5e-3}
+            self.check_double_backward_options = {'atol': 5e-3, 'rtol': 5e-1}
+
+    def generate_inputs(self):
+        return self.lhs, self.rhs
+
+    def forward_xp(self, inputs, xp):
+        if xp is numpy:
+            return xp.power(inputs[0], inputs[1]),
+        return xp.pow(inputs[0], inputs[1]),
```

Comment:
You can parameterize `is_module` to test both `chainerx.power` and `**` similar to this test https://github.com/chainer/chainer/blob/master/tests/chainerx_tests/unit_tests/routines_tests/test_math.py#L295